added support of styles, captions and width of columns

vovka-s opened the pull request.

vovka-s committed:
Merge pull request #1 from appropos/master

reintroduced captionStyleIndex

celsasser commented on index.js:
```diff
@@ -116,12 +120,24 @@ exports.execute = function(config, callback) {
         function(callback) {
+            return async.eachSeries(cols, function(col, callback) {
+                var colStyleIndex = col.styleIndex || 0;
+                var res = '<x:col min="' + cn + '" max="' + cn + '" width="' + col.width + '" customWidth="1" style="' + colStyleIndex + '"/>';
```
Would it make sense to default the col.width in the event that it's not specified so that it does go to undefined.

vovka-s replied:
fixed. Thanks!

celsasser commented:
This guy fixes col width and dates which would be good features in master.

Commit:
added default column width
added style for date. Use styleIndex=2 and set your local date format on 4th line of style.xml: <numFmt numFmtId="2" formatCode="dd.mm.yyyy;@"/>

vovka-s commented:
added default column width
added style for date. Use styleIndex=2 and set your local date format on 4th line of style.xml:
`<numFmt numFmtId="2" formatCode="dd.mm.yyyy;@"/>`

celsasser commented:
Looks good!

elwerene commented:
I somehow didn't get notified about this pull request. sorry! I will check it out on tuesday

elwerene commented:
Thanks and sorry for the long delay!

elwerene commented:
Please add yourself to the list of contributors! (by pull request)
